### PR TITLE
[CAS-1069] Docs - Documentation for suggestion list

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
[CAS-1069](https://stream-io.atlassian.net/browse/CAS-1069)

### 🎯 Goal

Restructure documentation for `MentionListView`

### 🛠 Implementation details

I only cut the images for the initial documentation. I don't see space to add more documentation as this view is currently not customization and we will add this option in the future. 

### 🎨 UI Changes

| Before | After |
| --- | --- |
| Screenshot 2021-06-16 at 15 44 10 | Screenshot 2021-06-16 at 15 43 49 |

### 🧪 Testing

Read the documentation.

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- ~[ ] Changelog is updated with client-facing changes~
- ~[ ] New code is covered by unit tests~
- [x] Comparison screenshots added for visual changes
- [x] Affected documentation updated (CMS, cookbooks, tutorial)
- [x] Reviewers added

### 🎉 GIF

giphy (1)

### CODE REVIEW

**Comment by adasiewiczr** [comment]
I would restructure this page to match `Channel List` which looks really nice.
What do you think about:
1. Removing `Overview` title and leaving its content directly below title
2. Combining `Adding MentionListView` and `Binding With ViewModel` into `Usage` and swapping with Action Handling section
3. Renaming `Handling Mention List View Actions` to `Handling Actions`?

File: docusaurus/docs/Android/03-ui/04-components/06-mention-list-view.md
Comment (bychkovdmitry) [inline docusaurus/docs/Android/03-ui/04-components/06-mention-list-view.md:33]:
```suggestion
We recommend using this view with its ViewModel, which configures the view and makes it fully usable. 
```

**Review by adasiewiczr — APPROVED** [review]
LGTM but let's wait for a second review from @nikomancy or @zsmb13

**Review by bychkovdmitry — APPROVED** [review]

**Pull request merged**
